src/aiq/tool/mcp/mcp_tool.py

```diff
@@ -33,8 +33,12 @@ class MCPToolConfig(FunctionBaseConfig, name="mcp_tool_wrapper"):
     function.
     """
     # Add your custom configuration parameters here
-    url: HttpUrl = Field(description="The URL of the MCP server")
+    url: HttpUrl | None = Field(default=None, description="The URL of the MCP server (for SSE mode)")
     mcp_tool_name: str = Field(description="The name of the tool served by the MCP Server that you want to use")
+    client_type: str = Field(default="sse", description="The type of client to use ('sse' or 'stdio')")
+    command: str | None = Field(default=None, description="The command to run for stdio mode (e.g. 'mcp-server')")
+    args: list[str] | None = Field(default=None, description="Additional arguments for the stdio command")
+    env: dict[str, str] | None = Field(default=None, description="Environment variables to set for the stdio process")
```

Comment:
Should have validation that the stdio properties are exclusively set from the SSE properties.

Reply:
Ack